exchange/exchange-ps/exchange/Set-AppRetentionCompliancePolicy.md

```diff
@@ -375,11 +379,15 @@ Accept wildcard characters: False
 ```
 
 ### -DeletedResources
-The DeletedResources parameter specifies the deleted mailbox or mail user to add as an exclusion to the respective location list. Use this parameter with the AddTeamsChatLocationException parameter for deleted mailboxes or mail users that needs to be excluded from a Teams only retention policy.
+The DeletedResources parameter specifies the deleted mailbox or mail user to add as an exclusion to the respective location list. Use this parameter with the AddTeamsChatLocationException parameter for deleted mailboxes or mail users that need to be excluded from a Teams only retention policy.
 
 A valid value is a JSON string. Refer to the Examples section for syntax and usage examples of this parameter.
 
-For information on the inactive mailbox scenario, see [Learn about inactive mailboxes](https://learn.microsoft.com/purview/inactive-mailboxes-in-office-365).
+For information about the inactive mailbox scenario, see [Learn about inactive mailboxes](https://learn.microsoft.com/purview/inactive-mailboxes-in-office-365).
+
+**CAUTION**: This parameter uses the SMTP address of the deleted mailbox or mail user, which might also be specified for other mailboxes or mail users. If you use this parameter without first taking additional steps, other mailboxes and mail users with the same SMTP address in the retention policy will also be excluded. To check for additional mailboxes or mail users with the same SMTP address, use the following command and replace *user@contoso.com* with the SMTP address to check: `Get-Mailbox -RecipientTypeDetails UserMailbox,SharedMailbox | Where-Object { $_.EmailAddresses -contains "SMTP:user@contoso.com" } | Select-Object DisplayName, EmailAddresses, Description, Alias, OrganizationalUnit`
+
+To prevent other mailboxes or mail users with the same SMTP address from being excluded, put the mailbox on [Litigation Hold](https://learn.microsoft.com/purview/ediscovery-create-a-litigation-hold) before you run the command with the DeletedResources parameter.
```

Comment:
"to prevent **active** mailbox or mailuser..."
An object that is already deleted cannot be re-added from customer side to hold. They can add to hold active users.